node/builder.go

```diff
@@ -163,12 +164,20 @@ func (nc *Builder) build(ctx context.Context) (*Node, error) {
 		return nil, err
 	}
 
+	// set up processor
+	var processor consensus.Processor
+	if nc.Rewarder == nil {
+		processor = consensus.NewDefaultProcessor()
+	} else {
+		processor = consensus.NewConfiguredProcessor(consensus.NewDefaultMessageValidator(), consensus.NewDefaultBlockRewarder(), builtin.DefaultActors)
```

Comment:
This looks like a copy error where the second argument should be nc.Rewarder.  If not there is something weird going on we should try to fix up.

Reply:
Good catch!